### DISCUSSION TITLE
Top Legend of the charts always placed on right

### DISCUSSION THREAD

**Discussion by vasu-ram** [apache/superset#27387] [Ideas]
### Bug description

Legend of the charts always placed on Top right, It would be great to keep, Top middle or left 

### How to reproduce the bug

Create a chart (line, bar.....)
Select values for x-axis and y-axis 
Select Legend option on customise tab
Try to bring the legend to centre
Keep only the values, remove or keep 

### Screenshots/recordings

https://github.com/apache/superset/assets/55632570/6a09fc34-d447-47c0-bc77-8a297ea46b5d

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [X] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by rusackas** [comment]
Have you checked ECharts to see if they support this? I assume so. It would just be adding an extra "dynamic control" for Left/Center/Right when the Top or Bottom choices are selected. As long as the defaults for the new control match the current situation, it doesn't seem like anyone ought to mind.

**Comment by rusackas** [comment]
This is more of a feature request than a bug though, so I'll convert it to a Discussion.
